src/ServiceControl.Monitoring.AcceptanceTests/Tests/When_querying_queue_length_data.cs

```diff
@@ -25,42 +25,26 @@ public class When_querying_queue_length_data : NServiceBusAcceptanceTest
                 .WithEndpoint<Receiver>()
                 .Done(c =>
                 {
-                    c.Response = GetRawMetrics();
-                    return c.Response != null && c.Response.Contains("Receiver@MyMachine");
+                    c.Response = GetRawMetrics(httpClient);
+                    return c.Response != null && c.Response.Contains("10");
                 })
                 .Run();
 
             Assert.IsNotNull(context.Response);
 
-            var expected = new JObject
-            {
-                {
-                    "NServiceBus.Endpoints", new JArray(
-                        new JObject
-                        {
-                            {
-                                "Receiver@MyMachine",
-                                new JObject
-                                {
-                                    {
-                                        "Count", 10
-                                    }
-                                }
-                            }
-                        }
-                    )
-                }
-            };
+            var result = JObject.Parse(context.Response);
```

Comment:
I think that this test was meant to play are role of a smoke test rather than testing the schema of the response. That being said I think it should be replaced by a test that calls `monitored-endpoints` as `metrics/queue-length` is not used at all.